source/architecture.rst

```diff
@@ -1,230 +1,245 @@
-applying rules over the API calls like traffic rate limitation and access control.
+The Kong API Gateways is used as the entry point for applications and external
+services to reach the services that are internal to the dojot platform,
+resulting in multiple advantages like, for instance, single access point and
+ease when applying rules over the API calls like traffic rate limitation and
+access control.
 
 Management Application
 **********************
 
-Web Application responsible for providing responsive interfaces to manage the *dojot* platform, including
-functionalities like:
+Web Application responsible for providing responsive interfaces to manage the
+*dojot* platform, including functionalities like:
 
-* **User Profile Management**: define profiles and the API permission associated to those profiles
+* **User Profile Management**: define profiles and the API permission
+  associated to those profiles
 * **User Management**: Creation, Visualization, Edition and Deletion Operations
-* **Applications Management**: Creation, Visualization, Edition and Deletion Operations
-* **Device Models Management**: Creation, Visualization, Edition and Deletion Operations
-* **Devices Management**: Creation, Visualization (real time data), Edition and Deletion Operations
-* **Processing Flows Management**: Creation, Visualization, Edition and Deletion Operations
+* **Applications Management**: Creation, Visualization, Edition and Deletion
+  Operations
+* **Device Models Management**: Creation, Visualization, Edition and Deletion
+  Operations
+* **Devices Management**: Creation, Visualization (real time data), Edition and
+  Deletion Operations
+* **Processing Flows Management**: Creation, Visualization, Edition and
+  Deletion Operations
 
 Elastic Service Controller
 **************************
 
-This is a service specialized for cloud environments, that is capable of monitoring the utilization of the platform,
-being able to increase or decrease its storage and processing capacity in an dynamic and automatic fashion to adapt to
-the variability on the demand.
+This is a service specialized for cloud environments, that is capable of
+monitoring the utilization of the platform, being able to increase or decrease
+its storage and processing capacity in an dynamic and automatic fashion to
+adapt to the variability on the demand.
+
+This controller depends that the dojot platform services are horizontally
+scalable, as well as the databases must be clusterizable, which match with the
+adopted architecture.
 
-This controller depends that the dojot platform services are horizontally scalable, as well as the databases must
-be clusterizable, which match with the adopted architecture.
+Alarm Management
+****************
+
+This component is responsible for handling alarms generated by dojot's internal
+components, such as IoT agents, Device Manager, and so on.
+
+Image manager
+*************
+
```

Comment:
Maybe the image manage would be better off with some more verbose description

Reply:
I ran out of ideas 😄